cjs-builder/index.ts

```diff
@@ -3,6 +3,9 @@ import esbuild from 'esbuild';
 import { mkdir, readFile, writeFile } from 'node:fs/promises';
 import { dirname, join } from 'node:path';
 
+// list of packages that should be bundled to the codemod (e.g codemod internal utils)
+const EXTERNAL_DEPENDENCIES = ['jscodeshift', 'ts-morph'];
```

Comment:
hmm what about `@intuita-inc/filemod`?

Reply:
we only import types from it, so its fine